packages/ui/src/machines/authenticator/signUp.ts

```diff
@@ -281,12 +275,12 @@ export function createSignUpMachine({ services }: SignUpMachineOptions) {
               'event',
               event
             );
+            const { nextStep } = event.data as Auth.ConfirmSignUpOutput;
+            const { signUpStep } = nextStep;
 
             if (context?.intent === 'confirmSignUp') {
```

Comment:
Does this still resolve to `true` at some point?

Reply:
I think at some point `isSignUpComplete` will resolve to true